utils_cv/detection/dataset.py

```diff
@@ -282,10 +285,21 @@ def __init__(
         # create training and validation data loaders
         self.init_data_loaders()
 
+    def _verify_data(self) -> None:
```

Comment:
[minor] add better description of what 'valid' means. or better yet, rename the function to be more descriptive.

Reply:
@jiata Thanks! Will address your comments and then re-request review. Also added some new visualization functionality today, which I need to write tests for before the next review.

Reply:
Added more text to hopefully make this more clear.